docker-compose-tests.yaml

```diff
@@ -5,12 +5,35 @@ networks:
     name: itu-minitwit-network
 
 services:
+  minitwit-database:
+    image: postgres:16-alpine
+    container_name: psql
+    networks:
+      - main
+    environment:
+      POSTGRES_USER: minitwit
+      POSTGRES_DB: minitwit_test
+      POSTGRES_PASSWORD: '1234'
+    ports:
+      - 5432:5432
+    healthcheck:
+      test: ["CMD-SHELL", "pg_isready"]
+      interval: 5s
+      timeout: 5s
+      retries: 5
+
   minitwit:
-    image: ${DOCKER_USERNAME}/minitwit
+    build:
+      context: .
+      dockerfile: Dockerfile
```

Comment:
Why do we not use the one from Dockerhub?

Reply:
Leftover from testing.. good find